llama_stack/providers/remote/memory/chroma/chroma.py

```diff
@@ -161,6 +161,14 @@ async def query_documents(
     ) -> QueryDocumentsResponse:
         index = self.cache.get(bank_id, None)
         if not index:
-            raise ValueError(f"Bank {bank_id} not found")
+            # if not in cache, try to get from chroma directly
```

Comment:
this does not seem to be the only place this should be done. see the `_get_and_cache_index()` methods we have in weaviate / qdrant. we need a similar one here in this class and  it should be used everywhere.

Reply:
done.